api/v1alpha1/componentplan.go

```diff
@@ -30,15 +30,21 @@ const (
 const (
 	ComponentPlanTypeSucceeded ConditionType = "Succeeded"
 	ComponentPlanTypeApproved  ConditionType = "Approved"
-	ComponentPlanTypeInstalled ConditionType = "Installed"
+	ComponentPlanTypeActioned  ConditionType = "Actioned"
```

Comment:
explain this `Actioned` condtion type

Reply:
A **clearer** name for what has been operated may be needed here...
The specifics of the operation will be described by `ConditionReason`, including:
- WaitDo
- Install: Installing / InstallSuccess / InstallFailed
- Uninstall: UnInstalling / UnInstallSuccess / UnInstallFailed
- Upgrade: Upgrading / UpgradeSuccess / UpgradeFailed

one example will be:
```yaml
  status:
    conditions:
    - lastTransitionTime: "2023-07-16T07:25:44Z"
      reason: ""
      status: "True"
      type: Approved
    - lastTransitionTime: "2023-07-16T07:26:14Z"
      reason: Upgrading
      status: "False"
      type: Actioned
    - lastTransitionTime: "2023-07-16T07:25:44Z"
      reason: ""
      status: "False"
      type: Succeeded
```

Reply:
👌  we should explain `CondtionType` in our document later.

Reply:
will track in #119